src/ZiggyCreatures.FusionCache/FusionCacheEntryOptions.cs

```diff
@@ -63,6 +65,16 @@ public FusionCacheEntryOptions(TimeSpan? duration = null)
 	/// </summary>
 	public TimeSpan Duration { get; set; }
 
+	/// <summary>
+	///  Gets the Microsoft.Extensions.Primitives.IChangeToken instances which cause the cache entry to expire.
```

Comment:
Utilizing a `<see cref="..."/>` tag here will enable consumers to navigate to the specified type.